stock_landed_costs_auto_create/models/purchase_order.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+from odoo.tests import Form
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    landed_costs_ids = fields.One2many(
+        comodel_name="stock.landed.cost",
+        inverse_name="purchase_id",
+        string="Landed Costs",
+    )
+
+    def _create_stock_landed_cost_extra(self):
+        lc_form = Form(
```

Comment:
I don’t think you should be using a `Form` here, as that is a test resource… Maybe just use the `create()` method.

Reply:
I have used `Form` here with the objetivo that the behavior is the same every time. A use case of this would be this: https://github.com/OCA/stock-logistics-workflow/pull/890

Reply:
I understand but, as I mentioned, it is a test resource used to replicate the "real" behavior, but it is not meant for this type of situations... Check the documentation in https://github.com/odoo/odoo/blob/13.0/odoo/tests/common.py#L1461 for more information...

Reply:
Ok, it's already changed. Thanks.